### PR TITLE
Remove google analytics

### PR DESCRIPTION
**Pull request by rezabekf**
*Description of changes:*
This PR removes google analytics from the hugo website.

### CODE REVIEW

File: workshop/config.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metaDataFormat = "yaml"
 defaultContentLanguageInSubdir= true
 
 uglyurls = false
-googleAnalytics = "UA-151248716-1"
+#googleAnalytics = "UA-151248716-1"
```
Comment (HieronymusLex) [inline workshop/config.toml:12]: why not just delete rather than comment? it's in the git history anyways

Comment (HieronymusLex) [reply workshop/config.toml:12]: shouldn't the script be removed from the template as well?

Comment (toksvaeth) [reply workshop/config.toml:12]: > shouldn't the script be removed from the template as well?

It's [built-in functionality](https://gohugo.io/templates/internal/) for Hugo.  I don't think we need to fork it to remove the unused templates.

Comment (rezabekf) [reply workshop/config.toml]: Thanks Nick, line has been removed.

**Review by ctd — APPROVED** [review]

File: workshop/config.toml (hunk 1 above)
Comment (rezabekf) [reply workshop/config.toml]: @toksvaeth footer has been updated.

**Pull request merged**